.github/workflows/openqa.yml

```
@@ -20,9 +20,16 @@ jobs:
     container:
       image: registry.opensuse.org/devel/openqa/containers/tumbleweed:client
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Trigger and monitor openQA test
-        run: >
+        id: openqa_schedule
+        run: |
+          if [[ "x$OPENQA_API_KEY" == 'x' && "x$OPENQA_API_SECRET" == 'x' ]]; then
```

That thing with the `x` is a really old bash workaround that shouldn't be necessary anymore. Also you can just use `-z`:
```suggestion
          if [[ -z "$OPENQA_API_KEY" && -z "$OPENQA_API_SECRET" ]]; then
```

---

I think what you mean is the "x" is a work-around for when bash is not being used. The `[[` is bash-specific which is why it's not needed. Empty strings are safe to compare here.